steveify1/nrc-hr-backend
# chore(readme):update README.md

#### What does this PR do?
update README.md
#### Description of Task to be completed?
* update README.md
#### How should this be manually tested?
* clone the repository
* cd into the project directory
* check the `README.md` file for update
#### Any background context you want to provide?
n/a
#### What are the relevant pivotal tracker stories?
#25
#### Screenshots (if appropriate)
#### Questions:
